packages/manager/src/queries/cloudpulse/services.ts

```diff
@@ -19,6 +23,15 @@ export const useGetCloudPulseMetricDefinitionsByServiceType = (
   });
 };
 
+export const useCloudPulseServices = () => {
```

Comment:
add all the queries inside the query factory. 
ref: https://github.com/linode/manager/blob/develop/packages/manager/src/queries/cloudpulse/queries.ts